utils/build/docker/python/flask.py

```diff
@@ -104,6 +108,24 @@ def identify_propagate():
     return Response("OK")
 
 
+@app.route("/dbm")
+def dbm():
```

Comment:
can you add in https://github.com/DataDog/system-tests/tree/main/docs/weblog the expected behavior of this endpoint ?